data/remnant/remnant missions.txt

```diff
@@ -2208,3 +2208,426 @@ mission "Remnant: Face to Maw 2B"
+			`	As he disappears from sight you hear a faint chime from your commlink, which displays a message indicating that two payments of <payment> have been allocated to your account. You recall that you didn't get a message after the last mission, so they must have combined them. You do find it odd, however, that they always seem to know exactly when you finish a delivery.`
+
+
+
+mission "Remnant: Crystal Research 7"
+	name "Deliver Data to <destination>"
+	description `A copy of the scanner data needs to be delivered to the researchers on <destination>.`
+	source "Viminal"
+	destination
+		government "Remnant"
+		not planet "Viminal"
+	to offer
+		has "Remnant: Crystal Research 6: done"
+	on offer
+		conversation
+			`You walk into the cafeteria and spot Aeolus halfway through an impressively large meal. He waves you over, and by the time you are seated a tray with a significantly smaller meal is waiting for you. While you have become accustomed to a few of the dishes, most of them are still very unappetizing.`
+			choice
+				`	"The Remnant sure like to eat."`
+				`	"Why do the Remnant eat so much?"`
+					goto foodvolume
+				`	"Maybe I should import some spices."`
+					goto spices
+				`	"What is our next step?"`
+					goto deliverdata
+			`	Aeolus chuckles and makes an amused gesture. "An army lives and dies by its food! A poorly fed soldier is but a short step away from being a deserter, or even a traitor. While we have had lean years in the past, these days we ensure everyone is well fed."`
+				goto deliverdata
+			label foodvolume
+			branch capitallicense
+				"license: Remnant Capital" = 1
+			`	He looks at you appraisingly. "You are perceptive, but there are some things you cannot learn yet. Ask us again when you have earned more of our trust."`
+				goto deliverdata
+			label capitallicense
+			apply "remnant symbionts info" += 1
+			`	He looks at you appraisingly "You are perceptive, and have earned our trust." He leans forward and holds up his hands. He pokes at the leather fingerless gloves he is wearing, and you realize that they extend up under his sleeves. You note that the leather does not actually look like any leather you have seen - it has fine scales and rough ridges on it. What stands out the most, however, is that it appears to pulse faintly.`
+			`	"The truth is, when we eat, we are eating for two. You will have to ask someone else how they came to be or how it works, but suffice to say that we have developed a symbiotic relationship with a variety of creatures we found here. These ones," He explains while holding up his hands, "have a knack for accuracy and durability. With them, I am more precise than I would otherwise be, and much tougher. In exchange, they feed through us and require regular care and attention. Normally, this causes us no hardship, but if we do not eat enough, we will consume our reserves far faster than we normally would."`
```

Comment:
Ways to highlight this idea before this point:
* Mentioned the gloves as being something that all Remnant wear earlier on. E.g. have it be a part of their description in the very first mission. (Perhaps if the player refuses the blood test but doesn't run and then gets grabbed by somebody, there could be a mention of the texture of the glove. Perhaps the player notices the pulsating when touching the "glove.")
* There are various feats of great strength or agility that the Remnant show in other missions, though they aren't very directly highlighted, only being said to have occurred. Perhaps have the player react with surprise/shock to one such past event.
* We know the Remnant have camels. Perhaps in a previous mission or in the description of a planet/for a spaceport, mention something about these camels perhaps having some sort of symbiote on them, or simply mention some odd "growth" on them.

Reply:
Firstly, not all Remnant have the glove symbiont, although probably quite a few people in military or trades specialties do.

Secondly, this *is* supposed to be the early mission that gives the player the first hints. In the grand scheme of things, unless the player is very unlucky, this set of missions will be completed during the early part of the "Rising action" phase of the Remnant story. While veteran players with old pilots will encounter this mission after having acquired the capital license; most future pilots will encounter this before they have done the Salvage mission chain. (Note that the initial Remnant Crystal research mission requires "Remnant: Learn Sign 2: Done", whereas Salvage 1 requires "Remnant: Learn Sign Follow Up: Offered" So at the very least, the player has a 60 day window in which they might get the crystal research mission before they could possibly get the first salvage mission. As such, most players will get a teasing hint that yes, the fact that they eat so much is actually noteworthy and possibly important, but not something the Remnant will trust them with yet.)

Third, I added a bit to the Remnant first contact mission, as well as one of the salvage missions. The former is a note regarding quantity of food; and the later is noting that some of the Remnant the player is evacuating are meditating while an IV feeds into their glove. I suspect most people will assume it feeds through their glove into the person, but perceptive people might be curious as to whether it is literally actually feeding the glove.

I'm currently working on expanding something to do with one or more of the camels.